commands/regions/index.js

```diff
@@ -5,9 +5,16 @@ const co = require('co')
 
 function * run (context, heroku) {
   const sortBy = require('lodash.sortby')
+  const filter = require('lodash.filter')
```

Comment:
Fixed in https://github.com/heroku/heroku-apps/pull/117/commits/6cd7ccb70ebb0fd594008b14e51be42a2089fc6f.